app/views/characters/skills_select.js.erb

```diff
@@ -0,0 +1,2 @@
+$("#character_skills").empty()
+  .append("<%= escape_javascript(render :partial => 'skills', :locals => {:skills => @skills}) %>")
```

Comment:
Use the semicolon syntax instead of the rocket. I think you can change the render partial for just render something like:

```ruby
<%= j render 'skills', :locals => { skills: @skills } %>
```